### PR TITLE
add about.html page

### PR DESCRIPTION
**Pull request by Filimon523**
hey guys, I added about.html page. Please review my code. Thank You!!

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                    <p>Products we offer</p>
+                </div>
+                <div class="footer-section">
+                    <h3>Contact Us</h3>
+                    <p>How to reach us.</p>
+                </div>
+            </div>
+            <p>Copyright &copy; 2024 Amazon&trade;</p>
+        </footer>
+    </div>
```
Comment (RhstiSWE) [inline about.html:74]: pls add here products html

Comment (Filimon523) [reply about.html:74]: okay..

**Review by RhstiSWE — APPROVED** [review]
good job file

**Review by RhstiSWE — CHANGES_REQUESTED** [review]
good job

**Pull request merged**